context/app/static/js/components/workspaces/ConfirmDeleteWorkspacesDialog/ConfirmDeleteWorkspacesDialog.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import {
+  Box,
+  Button,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  Divider,
+  IconButton,
+  Stack,
+} from '@mui/material';
+import CloseRounded from '@mui/icons-material/CloseRounded';
+
+import { useSnackbarActions } from 'js/shared-styles/snackbars';
+import { SelectedItems } from 'js/hooks/useSelectItems';
+import { generateCommaList } from 'js/helpers/functions';
+
+import { MergedWorkspace } from '../types';
+
+interface ConfirmDeleteWorkspacesDialogProps {
+  dialogIsOpen: boolean;
+  handleClose: () => void;
+  handleDeleteWorkspace: (workspaceId: number) => Promise<void>;
+  selectedWorkspaceIds: SelectedItems;
+  workspacesList: MergedWorkspace[];
+}
+export default function ConfirmDeleteWorkspacesDialog({
+  dialogIsOpen,
+  handleClose,
+  handleDeleteWorkspace,
+  selectedWorkspaceIds,
+  workspacesList,
+}: ConfirmDeleteWorkspacesDialogProps) {
+  const { toastError } = useSnackbarActions();
+
+  const handleDeleteAndClose = () => {
```

Comment:
We should also add some tracking here.

Reply:
What would adding tracking look like here/what tools would that use?

Reply:
Actually we are already tracking this in the `useDeleteWorkspace` hook. Sorry for the false alarm.